### PR TITLE
Tested shopping cart functionality

### PR DESCRIPTION
**Pull request by ivan0dyatlyukkk**
image

### CODE REVIEW

File: src/test/java/org/diatliuk/bookstore/repository/cart/CartItemRepositoryTest.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD
+    )
+    @Sql(
+            scripts = DELETE_DEFAULT_USERS_AND_CARTS,
+            executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD
+    )
+    void findById_withValidId_returnsCartItem() {
+        CartItem actualCartItem = cartItemRepository.findById(TEST_VALID_CART_ITEM_ID).get();
+
+        assertNotNull(actualCartItem.getShoppingCart());
```
Comment (andrii-hoienko) [inline src/test/java/org/diatliuk/bookstore/repository/cart/CartItemRepositoryTest.java:59]: Are you sure that is it enough to assert that shopping cart is not null, maybe makes sence to check content of cart?And actually make sure that you have at least one tets which checks content of something that you receive

**Review by andrii-hoienko — CHANGES_REQUESTED** [review]
GJ! Let`s improve your solution

File: src/test/java/org/diatliuk/bookstore/repository/cart/CartItemRepositoryTest.java (hunk 1 above)
Comment (andrii-hoienko) [reply src/test/java/org/diatliuk/bookstore/repository/cart/CartItemRepositoryTest.java:59]: It is not resolved, at least in CartControllerTest you don`t have assert where content of cart is checked fullt

**Review by andrii-hoienko — CHANGES_REQUESTED** [review]
Comments are not fully resolved

File: src/test/java/org/diatliuk/bookstore/repository/cart/CartItemRepositoryTest.java (hunk 1 above)
Comment (ivan0dyatlyukkk) [reply src/test/java/org/diatliuk/bookstore/repository/cart/CartItemRepositoryTest.java:59]: You are completely right, sorry, I added it to the next method and forgot to add to this one

**Review by vovasalyha — APPROVED** [review]

**Pull request merged**